mods_cocina_mappings/mods_to_cocina_recordInfo.txt

```diff
@@ -148,6 +148,7 @@ NB: Standard may need to be applied on a value by value basis rather than to the
   <descriptionStandard>aacr</descriptionStandard>
   <recordContentSource authority="marcorg">CSt</recordContentSource>
   <recordCreationDate encoding="marc">180305</recordCreationDate>
+  <recordChangeDate encoding='iso8601'>20200718050001.0</recordChangeDate>

```

Comment:
is `0001.0` valid for seconds in iso18601?

Reply:
This is from actual data, but I double-checked the standard and .0 is valid for fractional seconds.